sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -58,12 +151,28 @@ should be used:
 # Test the SYCL compiler and runtime
 
 Run LIT testing using the command below after building SYCL compiler and runtime.
-
+Use one of these commands:
+### Linux:
 ```bash
 make -j`nproc` check-all
+make -j`nproc` check-llvm
+make -j`nproc` check-llvm-spirv
+make -j`nproc` check-clang
+make -j`nproc` check-sycl
```

Comment:
recently i tried "make all" then "make check-sycl" but check-sycl was not recognized; I got a tip from @alexbatashev that it was necessary to "make sycl-toolchain" before "make check-sycl".  I don't know if this is still needed.